docs/user-guide/processors.md

```diff
@@ -13,6 +13,11 @@ Processors are community packages that enable stylelint to lint the CSS within n
 
 ## PostCSS syntax
```

Comment:
Let's just remove this whole section for now. This is the `processors.md` file. Once we resolve the relationship between processors and syntax we can revisit this.